src/main/java/dev/j3fftw/litexpansion/machine/AdvancedSolarPanel.java

```diff
@@ -141,22 +146,39 @@ public int getCapacity() {
     @AllArgsConstructor(access = AccessLevel.PRIVATE)
     public enum Type {
 
-        ADVANCED(Items.ADVANCED_SOLAR_PANEL, ADVANCED_DAY_RATE, ADVANCED_NIGHT_RATE, ADVANCED_OUTPUT,
-            ADVANCED_STORAGE, new ItemStack[] {
+        BASIC(Items.BASIC_SOLAR_PANEL, BASIC_DAY_RATE, 0, BASIC_STORAGE, new ItemStack[] {
+            Items.CARBON_PLATE, Items.CARBON_PLATE, Items.CARBON_PLATE,
             Items.REINFORCED_GLASS, Items.REINFORCED_GLASS, Items.REINFORCED_GLASS,
-            Items.ADVANCED_ALLOY, SlimefunItems.SOLAR_GENERATOR_4, Items.ADVANCED_ALLOY,
+            Items.ELECTRONIC_CIRCUIT, Items.MACHINE_BLOCK, Items.ELECTRONIC_CIRCUIT
+        }),
+	    LOW_VOLTAGE(Items.LOW_VOLTAGE_SOLAR_PANEL, LOW_VOLTAGE_DAY_RATE, 0, LOW_VOLTAGE_STORAGE, new ItemStack[] {
+            Items.REINFORCED_GLASS, Items.REINFORCED_GLASS, Items.REINFORCED_GLASS,
+            Items.ADVANCED_ALLOY, Items.BASIC_SOLAR_PANEL, Items.ADVANCED_ALLOY,
+            Items.ADVANCED_CIRCUIT, Items.ADVANCED_MACHINE_BLOCK, Items.ADVANCED_CIRCUIT
+        }),
+	    MEDIUM_VOLTAGE(Items.MEDIUM_VOLTAGE_SOLAR_PANEL, MEDIUM_VOLTAGE_DAY_RATE, 0, MEDIUM_VOLTAGE_STORAGE, new ItemStack[] {
+            Items.REINFORCED_GLASS, Items.REINFORCED_GLASS, Items.REINFORCED_GLASS,
+            Items.ADVANCED_ALLOY, Items.LOW_VOLTAGE_SOLAR_PANEL, Items.ADVANCED_ALLOY,
+            Items.ADVANCED_CIRCUIT, Items.ADVANCED_MACHINE_BLOCK, Items.ADVANCED_CIRCUIT
+        }),
+	    HIGH_VOLTAGE(Items.HIGH_VOLTAGE_SOLAR_PANEL, HIGH_VOLTAGE_DAY_RATE, 0, HIGH_VOLTAGE_STORAGE, new ItemStack[] {
```

Comment:
Remove the 0

Reply:
Wont that break the thing below it? its looking for 4 things and will only get 3

Reply:
Make a new 1 that accepts 3